scripts/deploy_assisted_service.sh

```diff
@@ -131,6 +137,8 @@ else
     wait_for_url_and_run ${SERVICE_BASE_URL} "spawn_port_forwarding_command $SERVICE_NAME $SERVICE_PORT $NAMESPACE $NAMESPACE_INDEX $KUBECONFIG minikube undeclared $SERVICE_INTERNAL_PORT"
 
     if [ "${DEBUG_SERVICE}" == "true" ]; then
+        # Close the minikube registry service port forwarding
+        pkill -f "port-forward"
```

Comment:
Why would we need to do this?

Reply:
cleanup

Reply:
Wouldn't that kill other port-fowarding jobs running on the machine?

Reply:
After adding the registry to the xinetd,  it's not relevant any more